### PR TITLE
cva6/dev to master

### PR DESCRIPTION
**Pull request by MikeOpenHWGroup**
CVA6 team: it would be good to have the latest additions on `cva6/dev` merged up to the master branch.

### CODE REVIEW

**Comment by JeanRochCoulon** [comment]
@zchamski, two actions need to be executed: approve the cva6/dev and master PR AND (more impacting for you) the merge between master and cva6/dev.

**Review by yanicasa — COMMENTED** [review]
Ok for gitlab-CI part

**Review by JeanRochCoulon — APPROVED** [review]
LGTM

File: cva6/sim/Makefile [hunk 1]
```diff
@@ -25,11 +25,24 @@ FLIST_TB   := $(CVA6_TB_DIR)/Flist.cva6_tb
...
+tool_path       ?=
+isscomp_opts    ?=
+issrun_opts     ?=
+isspostrun_opts ?=
+log             ?=
+variant         ?=
+
+TESTNAME := $(shell basename -s .o $(elf))
+
+ifeq ($(isspostrun_opts), "")
```
Comment (zchamski) [inline cva6/sim/Makefile:39]: If 'isspostrun_opts' is empty, the invocation of 'grep' will run forever.  Change the condition from 'ifeq' to 'ifneq' (this way the code will read more naturally than if the true and false branches were swapped.)

**Review by zchamski — APPROVED** [review]

**Comment by yanicasa** [comment]
FIXES https://github.com/openhwgroup/core-v-verif/issues/1296

**Pull request merged**